index.fr.md

```diff
@@ -0,0 +1,226 @@
+-   [Demandez : Intégrer l'accessibilité tout au long de la conception](http://www.uiaccess.com/accessucd/overview.html) fournit de l'aide sur l'intégration de l'accessibilité tout au long de la conception de sites Web et d'autres produits. Le chapitre [Test d'utilisation pour évaluer l'accessibilité](http://www.uiaccess.com/accessucd/ut.html) comprend les différentes phases suivantes :
+    -   Organiser un test d'utilisation pour évaluer l'accessibilité – [déterminer les caractéristiques des participants](http://www.uiaccess.com/accessucd/ut_plan.html#characteristics), [recruter des participants](http://www.uiaccess.com/accessucd/ut_plan.html#recruiting), fournir une compensation
+    -   Préparer le test d'utilisation pour évaluer l'accessibilité – préparer des supports de test, garantir l'accessibilité des installations, mettre en place et tester les technologies d'assistance, mener un test pilote, utiliser des [méthodes de screening](http://www.uiaccess.com/accessucd/screening.html)
+    -   Mener un test d'utilisation pour évaluer l'accessibilité – [interagir avec des personnes handicapées](http://www.uiaccess.com/accessucd/ut_conduct.html#interacting), préparer la pièce où aura lieu le test
+    -   [Créer un rapport à partir du test d'utilisation pour évaluer l'accessibilité](http://www.uiaccess.com/accessucd/ut_report.html) – distinguer les problèmes d'accessibilité des problèmes d'utilisation, tirer des conclusions, donner des informations sur les personnes handicapées
+-   [Livre blanc : mener des évaluations incluant les utilisateurs avec des personnes handicapées](https://web.archive.org/web/20160603030517/http://www-03.ibm.com/able/resources/userevaluations.html)
+-   De nombreux livres, articles, conférences, et autres ressources traitent des différentes méthodes pour évaluer l'utilisation. Ces ressources comprennent divers types de tests d'utilisation, des conceptions de tests, des protocoles de développement de tests, dont des questionnaires, des tâches, la collecte de données ; la réalisation de tests pilotes ; et le nombre de participants à inclure dans un test d'utilisation. Par exemple : [Les tests d'utilisation démystifiés](http://www.alistapart.com/articles/usability-testing-demystified/), [Des exemples de plans et de formulaires de test tirés de "The Handbook of Usability Testing"](http://www.wiley.com/WileyCDA/WileyTitle/productCd-0470185481,descCd-DOWNLOAD.html), et de [Rocket Surgery Made Easy: The Do-It-Yourself Guide to Finding and Fixing Usability Problems](http://www.sensible.com/rocketsurgery/index.html) avec un exemple de test d'utilisation, de scripts et de formulaires en ligne.
```

Comment:
- conference presentations : "supports de conférences" rather than "conférences" ?
- usability evaluation techniques : "différentes méthodes pour évaluer l'ergonomie" ?
- usability testing (used many times) : "tests d'ergonomie"